core/templates/pages/exploration-editor-page/translation-tab/translator-overview/translator-overview.directive.html

```diff
@@ -57,7 +57,7 @@
     margin: 7px 0 7px 15px;
     width: 350px;
   }
-  .oppia-translation-language-selector {
+  .oppia-translation-language-selectoror {
```

Comment:
Copy-paste mistake, here and in a bunch of places elsewhere. 

Try to **never** copy-paste large-scale changes like this -- instead, make the changes manually and use grep to see if you have missed anything. Also, carefully proofread your changes as soon as (or ideally before) you submit them to catch things like this. You must **always** conduct a human review of each changed line somewhere in the process.

Reply:
Yikes - my bad

Reply:
Done